mmv1/products/dialogflowcx/api.yaml

```diff
@@ -111,3 +111,127 @@ objects:
         name: 'enableSpellCorrection'
         description: |
          Indicates if automatic spell correction is enabled in detect intent requests.
+  - !ruby/object:Api::Resource
+    name: 'Intent'
+    base_url: "{{parent}}/intents"
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      An intent represents a user's intent to interact with a conversational agent.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/dialogflow/cx/docs'
+      api: 'https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.agents.intents'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: parent
+        url_param_only: true
+        input: true
+        description: |
+          The agent to create an intent for.
+          Format: projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>.
+      - !ruby/object:Api::Type::String
+        name: 'languageCode'
```

Comment:
This field currently doesn't do anything since it is represented as `url_param_only` but not included in the `base_url` or any other url override.

Reply:
you are right I removed url_param_only